src/backend/mailbox_manager/api/client/viewsets.py

```diff
@@ -183,6 +187,30 @@ def destroy(self, request, *args, **kwargs):
 
         return super().destroy(request, *args, **kwargs)
 
+    @action(detail=False, url_path="users", methods=["get"])
+    def get_available_users(self, request, domain_slug):
+        """API endpoint to search user to give them new access.
+        More filters and permission will be added soon.
+        """
+        domain = models.MailDomain.objects.get(slug=domain_slug)
+        abilities = domain.get_abilities(request.user)
+        if not abilities["manage_accesses"]:
+            raise exceptions.PermissionDenied()
```

Comment:
Should you consider adding a `get_available_users` to the abilities?

Reply:
User needs to list available users to manage access, so adding `get_available_users` to abilities would be a duplicate.